deny.toml

```diff
@@ -70,8 +70,8 @@ skip = [
     { name = "getrandom", version = "0.1" },
     { name = "libm", version = "0.1" },
     { name = "mach", version = "0.2" },
-    { name = "ndk", version = "0.2" },
-    { name = "ndk-glue", version = "0.2" },
+    { name = "ndk", version = "0.3" },
+    { name = "ndk-glue", version = "0.3" },
```

Comment:
```suggestion
```
This should be removed instead as ndk 0.3 is no longer present multiple times in the dependence tree